config/deploy/production.rb

```diff
@@ -1,38 +1,63 @@
-server '51.15.130.54',    user: 'root', roles: %w{cron}
-
-server '163.172.185.77',  user: 'root', roles: %w{app web}
-server '163.172.154.122', user: 'root', roles: %w{app web}
-server '163.172.161.163', user: 'root', roles: %w{app web}
-server '163.172.139.6',   user: 'root', roles: %w{app web}
-
-server '163.172.139.253', user: 'root', roles: %w{app worker}
-server '163.172.138.184', user: 'root', roles: %w{app worker}
-server '163.172.155.116', user: 'root', roles: %w{app worker}
-server '163.172.165.101', user: 'root', roles: %w{app worker}
-server '51.15.141.224', user: 'root', roles: %w{app worker}
-
-namespace :deploy do
-  task :restart do
-    on roles(:app) do |host|
-      execute "sleep #{rand(100)} && restart librariesio"
-    end
-  end
-
-  task :restart_web do
-    on roles(:web) do |host|
-      execute "sleep #{rand(100)} && restart librariesio"
-    end
-  end
-
-  task :stop do
-    on roles(:worker) do |host|
-      execute "stop librariesio"
-    end
-  end
-  task :start do
-    on roles(:worker) do |host|
-      execute "start librariesio"
-    end
-  end
-  after :publishing, :restart
+require "fog/google"
+require "json"
+google_creds = JSON.load(File.open('.google_creds.json'))
```

Comment:
Can this use the application default creds (which live in `~/.config/gcloud/application_default_credentials.json`)?

Reply:
Because this gets run from your local laptop that wouldn't work. If you have a GCP instance that you plan to run the capistrano commands from then yes, it would probably work.

Reply:
You can have ADC setup locally by running `gcloud auth application-default login` which we do for the rest of our setup

Reply:
I tried getting it to work with the credentials from ADC but it appears that the fog-google gem explicitly doesn't support that as of this moment. I attempted to get it working by following comments from an issue on their repository but it didn't work properly so I abandoned that effort until the fog-google implements a fix.